### PR TITLE
Adding locust test for http interface

### PR DESCRIPTION
**Pull request by jzeeck**

### CODE REVIEW

File: web/src/locust/locustfile.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
```
Comment (luuse) [inline web/src/locust/locustfile.py]: Python may not located at this place on all platforms and distributions. Using `#!/usr/bin/env python` is more portable.

File: web/src/locust/locustfile.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+from locust import HttpLocust, TaskSet, task
+import json
+import datetime
+
+def get_json_header():
+	return {"content-type":"application/json"}
+
+def get_epoch_timestamp():
+	delta = datetime.datetime.now() - datetime.datetime(1970, 1, 1)
+	return (long(delta.total_seconds()) * 1000) + ((delta.microseconds) / 1000)
```
Comment (luuse) [inline web/src/locust/locustfile.py]: One of these methods does not exist on Python 2 I believe. Can you double check?

File: web/src/locust/run.sh [hunk 3]
```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```
Comment (luuse) [inline web/src/locust/run.sh]: Setting `set -eu` and `set -o pipefail` is a [good idea](http://gfxmonk.net/2012/06/17/my-new-bash-script-prelude.html) in most bash scripts.

File: web/src/locust/locustfile.py (hunk 2 above)
Comment (jzeeck) [inline web/src/locust/locustfile.py]: I run python 2.7 on my machine. It is a requirement for locust... if you read the readme ;)

Comment (luuse) [inline web/src/locust/locustfile.py]: I don't think it works on 2.6 though? `return (time.clock()* 1000)`?

**Pull request merged**